scalatest/src/main/scala/org/scalatestplus/selenium/WebBrowser.scala

```diff
@@ -45,10 +45,9 @@ import org.openqa.selenium.support.ui.Select
 import org.scalatest.exceptions.TestFailedException
 import org.scalatest.exceptions.StackDepthException
 import org.openqa.selenium.JavascriptExecutor
-import org.scalatest.ScreenshotCapturer
+import org.scalatest.ScalaTestInternals.ScreenshotCapturer
 import org.scalatest.time.Nanosecond
```

Comment:
@cheeseng Shouldn't ScreenshotCapturer just be in org.scalatestplus.selenium? Is it used elsewhere?